assets/css/_vehicle_map.scss

```diff
@@ -1,3 +1,21 @@
+$map-component-z-index: (
+  "overlay-cutout": 800,
+  "control-layer": 1000,
+);
+
+.c-vehicle-map {
+  .c-street-view-switch {
+    // Same control layer as leaflet controls
+    z-index: map-get($map-component-z-index, "control-layer");
+  }
+
+  .c-cutout-overlay {
+    // Below controls, above map
+    z-index: map-get($map-component-z-index, "overlay-cutout");
+  }
+}
```

Comment:
@KaylaBrady Continuing from the [comment in `_map_page` on z-index maps](https://github.com/mbta/skate/commit/89d2fca7477460c21728350a6e6a59d651873f0b#r120826285)
I tried something else out in c1c2491bbdd4b49cdd51eb8e088a2222b8bf75cd shown above.

I the commit message has a bit of a description on this, but I think this is a better location since `<Map/>` has `allowStreetView` and therefore the street view control could show up on any map, not just the map page.